schema/galaxy_partner.js

```diff
@@ -0,0 +1,35 @@
+import galaxy from '../lib/loaders/galaxy';
```

Comment:
barebones schema for a response, looks like:

`{"id":41001,"name":"Vladimir Sokolov Studio Gallery","_links":{"self":{"href":"https://artsy-galaxy-staging.herokuapp.com/galleries/41001"}}}`

( I will most likely add some location stuff in the immediate future too)